docs/development/dependencies.md

```diff
@@ -126,6 +126,23 @@ context of poetry's venv, without having to run `poetry shell` beforehand.
 poetry install --extras all --remove-untracked
 ```
 
+## ...delete everything and start over from scratch?
+
+```shell
+# Stop the current virtualenv if active
+$ deactivate
+
+# Find the venv for poetry
+$ poetry env info --path
+# Remove all of the files from the current environment
+$ rm -rf $(poetry env info --path)
```

Comment:
Actually I think this can be replaced with

```suggestion
# Remove all of the files from the current environment
$ poetry env remove
```

?

Reply:
Trying it out locally, `poetry env remove` doesn't do anything on it's own.

```
$ poetry env remove
No virtualenv provided.
```

It seems like we could use `--all` but I'm not confident that this doesn't remove unrelated other venvs. Anyone with multiple venvs across different projects want to try it out? The goal is to only remove the venv in Synapse.

```
$ poetry env remove --help
[...]
Options:
      --all             Remove all managed virtual environments associated with the project.
```

Reply:
> Anyone with multiple venvs across different projects want to try it out? 

Sure. I have two synapse checkouts, `synapse-3` with two poetry envs and `synapse-2` with one poetry env.

```
dmr on titan in synapse-3  2022-09-15 19:08:49 ✔  $ poetry env list
matrix-synapse-RhMlrixz-py3.10
matrix-synapse-RhMlrixz-py3.11 (Activated)

dmr on titan in synapse-2   2022-09-15 19:09:02 ✔  $ poetry env list
matrix-synapse-7yaa6cKe-py3.10 (Activated)
```

Removing `--all` envs in synapse-3:

```
$ poetry env remove --all -v
Deleted virtualenv: /home/dmr/.cache/pypoetry/virtualenvs/matrix-synapse-RhMlrixz-py3.10
Deleted virtualenv: /home/dmr/.cache/pypoetry/virtualenvs/matrix-synapse-RhMlrixz-py3.11

$ poetry env list
[no output]
```

Afterwards, listing envs in synapse-2:

```
$ poetry env list
matrix-synapse-7yaa6cKe-py3.10 (Activated)
```

So I think `poetry env remove --all` is safe to use as a project-wide reset button.